setup.cfg

```diff
@@ -31,12 +31,14 @@ install_requires =
     invenio-records-resources>=5.0.0,<6.0.0
     lxml>=4.5.0
     PyYAML>=5.4.1
+    oaipmh-scythe @ git+https://github.com/ulbmuenster/invenio-oaipmh-scythe.git
 
 [options.extras_require]
 tests =
     pytest-black-ng>=0.4.0
     invenio-app>=1.4.0,<2.0.0
     invenio-db[postgresql,mysql]>=1.0.14,<2.0.0
+    pytest_httpserver>=1.0.10
```

Comment:
@slint had mentioned a few libraries in the [`RORHTTPReader` Pull Request](https://github.com/inveniosoftware/invenio-vocabularies/pull/315#discussion_r1608132835).
I guess the decision on which library to use does not matter too much (since it's test-only), but I was curious about how you chose this specific one.

Reply:
I did not look too deep for a library. `Response` (in my opinion) is heavily written for testings based on `requests`, while `oaipmh_scythe` is based on `httpx`. It seemed a good idea to use a library that directly hooks into the protocol, thus working independently from any library used to implement http calls.